### DISCUSSION TITLE
netdata

### DISCUSSION THREAD

**Discussion by fatmaAliGamal** [autobase-tech/autobase#422] [General]
how can use netdata at cluster with wal-g ??
and appear GUI of netdata at private subnet  at aws 
can you advise me?

**Comment by vitabaks** [comment]
Netdata currently does not provide a module for monitoring the state of wal-g backups.

The list of available modules can be found here: https://learn.netdata.cloud/docs/contributing/external-plugins/go.d.plugin

**Comment by fatmaAliGamal** [comment]
i need to monitor server of cluster

**Reply by vitabaks** [reply]
Specify 
`netdata_install: true`

for install Netdata on postgresql cluster nodes.

**Reply by fatmaAliGamal** [reply]
i make it but it made for each  node separated

**Reply by vitabaks** [reply]
Use https://www.netdata.cloud 

to consolidate all servers to one (or more) dashboard.
